tests/examplefiles/pov/abyss.pov

```diff
@@ -0,0 +1,835 @@
+// This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License.
```

Comment:
This is unfortunately problematic because Pygments is BSD-licensed. I don't think we can accept tests under CC-BY-SA.

Reply:
I see. I removed the licensed code and replaced it with a short snippet from my own works hopefully covering the cases for `.x`, logic operators and float parsing.